src/inference/src/dev/preprocessing/preprocessing.cpp

```diff
@@ -80,18 +82,18 @@ bool ngraph::pass::AddPreprocessing::run_on_model(const std::shared_ptr<ngraph::
                 i += meanImage->size();
             }
 
-            meanMap[it.first] = ngraph::opset3::Constant::create(ngraph::element::f32, shape, meanImageData);
+            meanMap[it.first] = ov::op::v0::Constant::create(ov::element::f32, shape, meanImageData);
         }
     }
 
-    ngraph::pass::Manager manager(get_pass_config());
-    auto preproc = manager.register_pass<ngraph::pass::GraphRewrite>();
+    ov::pass::Manager manager(get_pass_config());
+    auto preproc = manager.register_pass<ov::pass::GraphRewrite>();
 
     if (!scaleMap.empty()) {
-        preproc->add_matcher<ngraph::pass::AddStdScale>(scaleMap);
+        preproc->add_matcher<ov::pass::AddScale>(scaleMap);
     }
     if (!meanMap.empty()) {
-        preproc->add_matcher<ngraph::pass::AddMeanSubtract>(meanMap);
+        preproc->add_matcher<ov::pass::AddMeanSubtract>(meanMap);
     }
 
     manager.run_passes(f);
```

Comment:
transformations from template plugin are not properly tested, so why not to use preprocessing API?